courseware/intro_data_science.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+title: Data Science with Real Data
+nav_order: 1
+parent: Courseware
+---
+
+# Data Science with _Real Data_
+
+Introductory Data Science and Machine Learning courses crave _real world_ datasets to enhance student interest and enrich their learning experience. However, identifying, accessing and preparing _real data_ can be a painstaking task. As a result, several foundational courses tend to rely on a similar subset of datasets, e.g. the [ISLR data package](https://cran.r-project.org/web/packages/ISLR/ISLR.pdf). We hope to demonstrate that Data Commons can help increase the diversity of _real world_ data used in such foundational courses taught across the world and enrich students' (and instructors') experience.
+
+We make available an (increasing) sample of data science course assignments developed around illustrating key concepts at an introductory college level. In addition to revolving around core data science ideas, we use _real world_ data provided by the Data Commons APIs with the aim of enhancing the pedagogical goals of each topic. Each assignment is implemented as a [python notebook](https://colab.research.google.com/). These notebooks are not teaching notes; they serve as self-contained templates for implementing, interpreting and/or analyzing a subset of core concepts. The entire assignment revolves around using some publicly available dataset, most often directly using the Data Commons APIs. 
+
+Each assignment notebook should ideally be adapted to suit the needs of your curriculum and serve the needs of a complete and coherent course. We intend for them to serve as examples (templates) for you to customize extensively. We encourage course instructors and teaching assistants to use different datasets (and variables) for each iteration of their course. Luckily, Data Commons makes this easy.
+
+All material is provided publicly and free of charge, under a Creative Commons License ([CC BY](https://creativecommons.org/licenses/by/4.0/)). If you end up finding any of this material useful and would like to be notified of updates, do drop us a line at courses@datacommons.org.
```

Comment:
Let's make the email address here a mailto: link as well

Reply:
Good catch! Implemented in new commit.